docker/Dockerfile.alpine

```diff
@@ -9,8 +9,8 @@ ENV TZ=Etc/UTC \
     LANG=C.UTF-8 \
     # Set default settings that may get overridden to empty values by
     # the entrypoint script, if not explicitly provided by the user
-    OIDC_SCOPE="openid profile email"
-
+    OIDC_SCOPE="openid profile email" \
+    BASE_HREF="/"
```

Comment:
Consider renaming this to `BASE_URL` or `BASE_PATH` so it's more in line with how the other options are worded (i.e. `API_BASE_URL`).

Reply:
BASE_PATH probably makes most sense, and would trigger anyone to put in the full URL. Updated